apps/music/manifest.webapp

```diff
@@ -1,21 +1,18 @@
 {
   "name": "Music",
   "description": "Gaia Music",
-  "type": "certified",
+  "type": "trusted",
+  "url": "http://music.hostedwebtid.es",
```

Comment:
This field shouldn't be here, but it might help us to know where we will host it